Containerfile

```diff
@@ -11,10 +11,12 @@ RUN pip install datascience \
     nb2pdf \
     PTable \
     pytest-custom-report \
+    scipy \
```

Comment:
I'm gonna have to ask for sources because: 

> 0 results for all repositories matching scipy

Outside of that, this is a new addition, so I'm not sure how much of the legit scipy environment is needed.

Reply:
nm, Looks like we're overriding the from tag in the build for juypyter base?

Reply:
Quick, ship it before it breaks again!